blocks/init/src/blocks/components/heading/components/heading-editor.js

```diff
@@ -1,28 +1,30 @@
 import React from 'react'; // eslint-disable-line no-unused-vars
+import classnames from 'classnames';
 import { __ } from '@wordpress/i18n';
 import { RichText } from '@wordpress/block-editor';
 
 export const HeadingEditor = (props) => {
+  const {
+    content,
+    styleAlign,
+    styleColor,
+    styleSize,
+  } = props.heading;
```

Comment:
Same as the button example, instead of `(props)` -> `({heading})` in the arrow function argument.